### DISCUSSION TITLE
Adding  Algorand to Wormhole

### DISCUSSION THREAD

**Discussion by Vinitowinit** [wormhole-foundation/wormhole#566] [Feature requests] [answered]
Along with current bridges in the ecosystem, it would be great if the wormhole can support Algorand as well.

The rise of Multichain projects demands a high need to have interoperability between chains. Algorand is planning to have strong growth this upcoming year with 500 million funding. Defi is going to grow in this space and I believe there will be major collaboration cross-chain especially among nonEVM L1s.

I would like to coordinate with Wormhole to build out an extension to the Algorand ecosystem. I was told to note this idea in the Discussion forum here on Github ( as directed in Discord )

**Comment by Flip-TechDev** [comment]
Would love to help.

**Comment by aadam-10** [comment, marked as answer]
Closing topic, Algorand has been added to the network by governance a while ago.
